x-pack/metricbeat/module/aws/ec2/ec2.go

```diff
@@ -157,7 +158,7 @@ func createCloudWatchEvents(getMetricDataResults []cloudwatch.MetricDataResult,
 	}
 
 	event.RootFields.Put("service.name", metricsetName)
-	event.RootFields.Put("cloud.provider", metricsetName)
+	event.RootFields.Put("cloud.provider", "aws")
```

Comment:
This is kind of a breaking change. Should we mention it in the changelog.

Reply:
Added into changelog, thank you for pointing this out!